---
title: 'BM424 workshop: infectious disease outbreak control'
author: "Morgan Feeney"
date: "24/02/2023"
output: 
  bookdown::html_document2:
    toc: true
    toc_float:
      toc_collapsed: false
    number_sections: false 
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(ggplot2)
library(knitr)

outbreak_deaths <- floor(runif(1, min = 10, max = 30))
city_name <- "Rosblane" 
hospital_name <- "North Shore" 
```

## Scenario D

Disaster has struck - hard on the heels of the COVID-19 pandemic, another disease outbreak has hit your fair city, `r city_name`. Residents are outraged and worried. There have already been `r outbreak_deaths` deaths.

The cause of the outbreak appears to be <i>Campylobacter jejuni</i>. (Please refer to the [WHO <i>Campylobacter</i> Fact Sheet](https://www.who.int/news-room/fact-sheets/detail/campylobacter) for more information about the pathogen.)

```{r Campy, out.width="50%", out.height="50%", echo=FALSE, fig.cap="Scanning electron microscopy (SEM) image of Campylobacter jejuni cells. Image credit: [CDC](https://phil.cdc.gov/Details.aspx?pid=5778)"}
knitr::include_graphics(rep("images/Campylobacter.jpg"))
```

Each member of your group will play one of the following roles as you attempt to tackle the disease outbreak and save your city:

1.	Provost (head of the city council)
2.	Representative from a national public health org. (Public Health Scotland)
3.	Epidemiologist (on secondment from Public Health Scotland)
4.	Director of the city's main hospital (`r hospital_name`)
5.	City planner/civil servant
6.	Microbiologist (working in one of the NHS Scotland clinical labs)
7.	Public communications expert (past and current sci comm campaigns)
8.  Doctor at a major surgery in `r city_name`
9.  Epidemiologist from the local university, an expert on <i>Campylobacter</i> transmission

You must work together, using the information that has been provided for you in the attached information packets, and decide what action(s) should be taken to control the outbreak currently plaguing your city, `r city_name`. Your actions should be evidence-based – use the peer-reviewed literature to decide on measures that will stop the pandemic in your scenario.

Your action points may include directives to gather more information (e.g., contact tracing of current cases, microbiological testing of food/water, etc.), or specific directives (e.g. public health measures, non-pharmaceutical interventions, etc.). Use the data/expertise of all group members. Be clear, concise, and specific.

You may choose to include mathematical modeling as part of your decision making (for example, using an [epidemic calculator](http://gabgoh.github.io/COVID/index.html) or other models); if you do so, you should provide details of your modelling and predictions when you submit your group's pro forma.

You should submit 3-5 specific actions to control the pandemic, using the [workshop 3 pro forma](/proformas/BM424_workshop3_proforma.docx) (also available on MyPlace). Submit this by noon, Friday March 1st, via the submission link on MyPlace. 

***

## Information Packet D1

You are the provost of `r city_name`, elected in `r sample(2012:2021, size=1)`. 

```{r city, echo=FALSE, fig.cap="Photograph of Rosblane city centre. Image credit: DALL-E"}
knitr::include_graphics(rep("images/Rosblane.png"))
```


The current population of `r city_name` (as of December 2022) is `r sample(150000:170000, size=1)`. The demographic data for the city is shown below.

```{r A1, fig.show="hold", out.width="50%", echo=FALSE, fig.cap="Rosblane demographics (Source: Office for National Statistics)"}

Pop <- c(sample(150000:170000, size=3, replace=TRUE), sample(170000:180000, size=3, replace=TRUE), sample(190000:170000, size=2, replace=TRUE), sample(150000:170000, size=2, replace=TRUE))
PopYears <- c(2012:2021)
population <- data.frame(Pop, PopYears)
ggplot(data = population, aes(x=PopYears, y=Pop)) + 
  geom_point() +
  geom_line() +
  labs(y = "Number of Rosblane residents (estimated)", x = "Year") +
  scale_y_continuous(limits=c(0, 200000)) + 
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())


Demog <- c((rep("Births", 10)), (rep("Deaths", 10)))
Num <- c(sample(1000:2000, size=10, replace=TRUE), sample(500:1500, size=8, replace=TRUE), sample(3000:5500, size=2, replace=TRUE))
DemogYears <- c(rep(2012:2021, 2))
birthdeath <- data.frame(Demog, Num, DemogYears)
ggplot(data = birthdeath, aes(x=DemogYears, y=Num, group=Demog, colour=Demog)) + 
  geom_point() +
  geom_line() +
  labs(y = "Number of births and deaths in Rosblane (estimated)", x = "Year", colour = "Demographics") +
  scale_y_continuous(limits=c(0, 5500)) + 
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + 
  scale_color_discrete(name="")
```
You were elected as a member of the popular "Peace and Prosperity" party and made a number of election promises regarding a "Green New Deal" for `r city_name`.

```{r A2, fig.cap="Voting intention by Rosblane residents (Source: YouGov polling)", echo=FALSE}

Party <- c((rep("New Hope", 10)), (rep("Wind and Solar Power", 10)), (rep("Peace and Prosperity", 10)), (rep("Rational Industry", 10)), (rep("Social Unity", 10)))
Polls <- c(sample(1:3, size=10, replace=TRUE), sample(8:14, size=10, replace=TRUE), sample(15:20, size=10, replace=TRUE), sample(5:10, size=10, replace=TRUE), sample(1:2, size=10, replace=TRUE))
PolYears <- c(rep(2012:2021, 5))
politics <- data.frame(Party, Polls, PolYears)
ggplot(data = politics, aes(x=PolYears, y=Polls, color=Party)) + 
  geom_point() +
  geom_line() +
  labs(y = "Voting Intention", x = "Year") + 
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())

Councillors <- sum(politics[politics$PolYears>2020, 2])
```

There are currently `r Councillors` elected councillors, representing the 5 main political parties in `r city_name` as follows:

```{r A3, echo=FALSE}

pol2021 <- (politics[politics$PolYears>2020, 2])
Parties <- c("New Hope", "Wind and Solar Power", "Peace and Prosperity", "Rational Industry", "Social Unity")

Councillors <- data.frame(Parties, pol2021)

kable(x = Councillors, 
      format = "html", 
      col.names = c("Political Party", "Number of Seats"))

```
<br><br>
The overall budget (expenditures) for the 2021-2022 financial year is £`r sample(400000000:600000000, size=1)`. A summary of expeditures by category for this financial year is shown below. 

```{r A4, fig.cap="Public expenditures by category for the 2021-2022 financial year (Source: Rosblane City Council)", echo=FALSE}

expenses <- data.frame(
  Category=c("Education", "Industry and Agriculture", "Social Care", "Transport", "Public order and safety", "Healthcare", "Housing", "Other"),
  value=c(sample(8:15, size=1), sample(10:15, size=1), sample(4:10, size=1), sample(5:8, size=1), sample(5:9, size=1), sample(20:28, size=1), sample(7:15, size=1), sample(1:4, size=1))
)

ggplot(expenses, aes(x="", y=value, fill=Category)) +
  geom_bar(stat="identity", width=1, color="white") +
  coord_polar("y", start=0) +
  theme_void() 
```


```{r emails, echo=FALSE}

clos1 <- c("Yours sincerely,", "Best wishes,", "Kind regards,", "All the best,", "Sincerely,", "Yours truly,", "Respectfully,")
email1 <- c("gldk", "aptr", "bhr", "tgh", "pbr", "js", "lucy", "amd", "oirl", "jim", "ygg", "oli", "end", "lulu", "rye", "att")
adj1 <- c("Disgraceful", "Terrible", "Horrible", "Devastating", "Sad", "Bloody")
emotion1 <- c("outraged", "upset", "extremely vexed", "so angry", "devastated")
sent1 <- c("As if COVID wasn't enough! Has this city not suffered enough, I ask you.", "This seems to be too much to bear on top of COVID, the cost of living crisis, and the war in Ukraine. It is just one tragedy after another.", "Too many people have already died from COVID and it seems completely cruel to have another pandemic strike our city again so soon.", "There seems to be no end to the suffering that pandemics inflict on our city in these days.", "It seems almost like the apocalypse must be upon us, with this pandemic following so close upon the heels of COVID19.", "Too many people have died and it is such a tragedy.")
wish1 <- c("hope", "hope very much", "wish", "expect", "sincerely hope")
state1 <- c("dreadful state of affairs", "horrible pandemic", "all of the poor people dying", "tragedy")
time1 <- c("immediately", "very soon", "at once", "as soon as possible")
name1 <- c("Anna", "Luke", "Siobhan", "Su", "Ailidh", "Fiona", "Graham", "Madison", "Eleanor", "Rodrigo", "Kieran", "Wilson", "Anika", "Rosemary", "Alise", "Gordon", "Roman", "Karen")
verb1 <- c("devastate", "obliterate", "ruin", "destroy", "crush")
business1 <- c("small coffee shop", "coffee shop", "cafe", "restaurant", "bookshop", "boutique clothing store", "records shop", "used bookstore", "toy store", "sweet shop")
business2 <- c("The Rosblane Shop", "Fraser & Co.", "Bella's", "Around the Corner", "Jewels")
adj2 <- c("nearly", "almost", "virtually", "almost literally", "practically")
conseq1 <- c("lost my business due to those pandemic lockdowns", "had to close because of all the draconian lockdowns", "went out of business", "went bankrupt", "had to shut our doors during COVID")
sent2 <- c("I hope you realise how costly the last pandemic was for millions of people across the country, and that we can't afford another one.", "You should be ashamed to think of all the businesses that have had to close because of the coronavirus measures this city took, and all of the people who lost their jobs.", "Businesses must be free to operate, not prosecuted by some nanny state.", "Things are bad enough already with the cost of living crisis, war in Ukraine, and Brexit, we can't afford another pandemic.", "I hope you realise that you need to put businesses first now - we were unfairly forced to bear the costs of the last pandemic.", "I hope you realise how much business we have all lost in recent years, and how much businesses in this city are struggling financially.")
subj1 <- c("Government Overreach", "Government Overstep", "Stop This Madness", "New Pandemic Measures")
state2 <- c("awful pandemic", "terrible pandemic", "new crisis", "new pandemic", "new disease")
conseq2 <- c("another lockdown", "a return to COVID rules", "face masks and gallons of hand sanitizer", "the draconian lockdowns you imposed during COVID", "restrictions that cripple our businesses")
wish2 <- c("fear", "dread", "abhor the idea", "hate the idea", "am afraid")
sent3 <- c("This last pandemic was just a typical example of what governments do when given a little too much power.", "Machiavelli said that power corrupts absolutely, and he was absolutely right - look at what happened during the last pandemic.", "I am sick and tired of government interference with my life, I have the right to go about my life without some do-gooder from a public health agency telling me that I need to wear a mask and can't visit my grandmother.", "The mental health consequences of the last pandemic were underappreciated, but very real - people suffered due to the lockdowns and children's development was affected by the wearing of face masks and the closing of schools.")
sent4 <- c("Please do not use this disaster as another opportunity for oppressive lockdowns and government control of our lives.", "You must not allow the city to suffer through another round of lockdowns, face-masks, and other punitive restrictions that destroy people's lives.", "I hope that you will do the right thing in this time of crisis and that I will be able to vote for you again in the coming elections with a clean conscience.", "Facemasks are just the tyranny of the minority over the majority, I hope that you will not permit them again on your watch.")
subj2 <- c("This Terrible Tragedy", "Another Terrible Loss")
rel1 <- c("my mother-in-law", "my aunt", "my auntie", "my niece", "my nephew", "my husband's best friend", "my wife's best friend", "my sister-in-law", "my brother-in-law", "my uncle", "my granddad", "my gran", "my nana", "my nana's best friend")
sent5 <- c("The family are all devastated as you can surely imagine.", "Words cannot begin to describe our loss.", "This is truly a tragedy.", "This is an unspeakable loss.", "The family are all devastated by this loss.", "You cannot begin to imagine the grief that we are feeling.", "This is especially terrible coming hard on the heels of all of our losses due to COVID-19.")
sent6 <- c("I don't understand why we haven't learned how to deal with pandemics better, after all our experiences with COVID-19, I don't understand how this could have happened.", "I really wish that our government had better learned its lessons from the COVID-19 pandemic and could have prevented this terrible tragedy.", "I hope that, considering how many other families have also lost loved ones, that you and your office will be doing everything in their power to halt this pandemic and prevent more needless deaths.", "I hope that you will do everything you can to stop this pandemic so that no other families need to suffer this pain.", "I hope that you will do everything you can to stop this pandemic so that no more lives will be lost.", "I am sure that you and your office are doing your best, but I hope that in the fullness of time, there will be a public inquiry into how this pandemic was handled and why so many families have lost loved ones. Especially coming so soon after the lessons that we learned during the COVID-19 pandemic, I would have thought that this tragedy could have been prevented.")
```

Your office has recently received a great deal of correspondence regarding the pandemic in `r city_name`. A representative sample of these e-mails is shown below.

***
<b>From:</b> `r sample(email1, 1)`_`r sample(1000:100000, size=1)`@gmail.com <br>
<b>To:</b> provost@`r city_name`.co.uk <br>
<b>Cc:</b>
<b>Subject:</b> This `r sample(adj1, 1)` Pandemic

Dear Provost,

I am `r sample(emotion1, 1)` to hear about this new pandemic in `r city_name`. `r sample(sent1, 1)` 

I `r sample(wish1, 1)` that your office is going to do something about `r sample(state1, 1)` `r sample(time1, 1)`.

`r sample(clos1, 1, replace=TRUE)`<br>
`r sample(name1, 1)`

***

<b>From:</b> `r sample(email1, 1)`_`r sample(1000:100000, size=1)`@gmail.com <br>
<b>To:</b> provost@`r city_name`.co.uk <br>
<b>Cc:</b> city-council@`r city_name`.co.uk <br>
<b>Subject:</b> Pandemic Disaster for Businesses

Dear Provost,

This new pandemic in `r city_name` has the potential to `r sample(verb1, 1)` our businesses. I run a `r sample(business1, 1)` on the high street and `r sample(adj2, 1)` `r sample(conseq1, 1)` due to COVID. 

`r sample(sent2, 1)` 

I `r sample(wish1, 1)` that your office is going to do something about `r sample(state1, 1)` `r sample(time1, 1)`, keeping in mind the economic interests of businesses in our city and the fact that we cannot afford `r sample(conseq2, 1)`.<br>

`r sample(clos1, 1, replace=TRUE)`<br>
`r sample(name1, 1)` (`r sample(business2, 1)`, proprietor)

***

<b>From:</b> `r sample(email1, 1)`_`r sample(1000:100000, size=1)`@gmail.com <br>
<b>To:</b> provost@`r city_name`.co.uk <br>
<b>Cc:</b> city-council@`r city_name`.co.uk <br>
<b>Subject:</b> `r sample(subj1, 1)`

Dear Provost,

I `r sample(wish2, 1)` that you and your office are going to use this new pandemic as yet another excuse for government overreach into citizens' lives. `r sample(sent3, 1)` 

`r sample(sent4, 1)` 

`r sample(clos1, 1, replace=TRUE)`<br>
`r sample(name1, 1)` (a concerned voter)

***

<b>From:</b> `r sample(email1, 1)`_`r sample(1000:100000, size=1)`@gmail.com <br>
<b>To:</b> provost@`r city_name`.co.uk <br>
<b>Cc:</b> city-council@`r city_name`.co.uk <br>
<b>Subject:</b> 

Dear Provost,

I have just learned that my `r sample(rel1, 1)` has passed away due to the new pandemic in `r city_name`. `r sample(sent5, 1)` 

`r sample(sent5, 1)` 

`r sample(sent6, 1)` 

`r sample(clos1, 1, replace=TRUE)`<br>
`r sample(name1, 1)` 

***


As Provost of `r city_name`, you are of course familiar with the work of the [Standing Committee on Pandemic Preparedness](https://www.gov.scot/groups/standing-committee-on-pandemic-preparedness/). You are also familiar with:

* The [UK Pandemic Preparedness plan](https://www.gov.uk/government/publications/uk-pandemic-preparedness/uk-pandemic-preparedness)

* The latest [COVID-19 health protection guidance](https://publichealthscotland.scot/our-areas-of-work/conditions-and-diseases/covid-19/covid-19-health-protection-guidance/#section-1) released by Public Health Scotland

* [Tackling antimicrobial resistance 2019–2024: The UK’s five-year national action plan](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1070263/UK_AMR_5_year_national_action_plan.pdf)

* The [UK One Health Report - Joint report on antibiotic use and antibiotic resistance, 2013–2017 ](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/921039/Ted_Final_version__1318703-v45-One_Health_Report_2019_FINAL-accessible.pdf)

* [Communicating risk in public health emergencies: A WHO guideline for emergency risk communication (ERC) policy and practice](https://apps.who.int/iris/bitstream/handle/10665/259807/9789241550208-eng.pdf?sequence=2&isAllowed=y)

 

***

## Information Packet D2

You are a representative from Public Health Scotland, sent to `r city_name` to help with the <i>Campylobacter jejuni</i> outbreak. 

Your briefing packet includes the following documents: 

* [Campylobacter guidance, data and surveillance (PHS)](https://www.hps.scot.nhs.uk/a-to-z-of-topics/campylobacter/)

* [Campylobacter Food Poisoning (Food Standards Scotland)](https://www.foodstandards.gov.scot/consumers/food-safety/foodborne-illness/campylobacter)

* [Campylobacter: Estimating the burden of gastrointestinal infection in Scotland using data linkage](https://www.foodstandards.gov.scot/downloads/HPS_-_Campylobacter_Data_Linkage_Report.pdf)

* Bessell, Paul R et al. “Geographic determinants of reported human Campylobacter infections in Scotland.” BMC public health vol. 10 423. 15 Jul. 2010, doi:10.1186/1471-2458-10-423

***

## Information Packet D3

You are an epidemiologist working for Public Health Scotland, sent to `r city_name` to help with the <i>Campylobacter</i> outbreak.

A team of scientists working together have in a PHS lab have isolated and cultured <i>Campylobacter jejuni</i> (following standard procedures) from a number of patient samples taken during this outbreak; isolated <i>Campylobacter jejuni</i> DNA from these cultures; sequenced this DNA using an Illumina MiSeq platform; assembled the short paired-end reads; and compared these assemblies to the extant <i>Campylobacter jejuni</i> genome sequences available in the NCBI databases. The results of these experiments are summarised in the table below. 

<table>
<table border="1">
  <tr>
    <th>Patient  </th>
    <th>NCBI accession of closest database match</th>
  </tr>
  <tr>
    <td>1</td>
    <td>GCA_001686905.1</td>
  </tr>
  <tr>
    <td>2</td>
    <td>GCA_001299565.1</td>
  </tr>
  <tr>
    <td>3</td>
    <td>GCA_001686905.1</td>
  </tr>
  <tr>
    <td>4</td>
    <td>GCA_001686905.1</td>
  </tr>
  <tr>
    <td>5</td>
    <td>GCA_000835285.1</td>
  </tr>
  <tr>
    <td>6</td>
    <td>GCA_000835285.1</td>
  </tr>
    <tr>
    <td>7</td>
    <td>GCA_000835285.1</td>
  </tr>
  <tr>
    <td>8</td>
    <td>GCA_000835285.1</td>
  </tr>
  <tr>
    <td>9</td>
    <td>GCA_000835285.1</td>
  </tr>
    <tr>
    <td>10</td>
    <td>GCA_000835285.1</td>
  </tr>
</table>  
<br><br>
Your briefing packet also includes the following documents: 

* [<i>Campylobacter jejuni/coli</i> pubMLST](https://pubmlst.org/organisms/campylobacter-jejunicoli)

* Bessell, Paul R et al. “Using sequence data to identify alternative routes and risk of infection: a case-study of campylobacter in Scotland.” BMC infectious diseases vol. 12 80. 1 Apr. 2012, doi:10.1186/1471-2334-12-80

* Llarena, Ann-Katrin et al. “Whole-Genome Sequencing in Epidemiology of Campylobacter jejuni Infections.” Journal of clinical microbiology vol. 55,5 (2017): 1269-1275. doi:10.1128/JCM.00017-17

* Joseph, Lavin A et al. “Comparison of Molecular Subtyping and Antimicrobial Resistance Detection Methods Used in a Large Multistate Outbreak of Extensively Drug-Resistant Campylobacter jejuni Infections Linked to Pet Store Puppies.” Journal of clinical microbiology vol. 58,10 e00771-20. 22 Sep. 2020, doi:10.1128/JCM.00771-20

***

## Information Packet D4

You are the director of `r city_name`'s main hospital, the `r hospital_name`, which has 550 beds across 9 different wards and 10 operating theatres. 


```{r hospital, echo=FALSE, fig.cap="The North Shore hospital in Rosblane. Image credit: [DALL-E"}
knitr::include_graphics(rep("images/Queens-College.png"))
```

The annual operating budget for financial year 2021-2022 was £1.5 million, and average expenditures are shown by category in the figure below. 

```{r A5, fig.cap="Hospital expenditures by category for financial year 2021-2022 (Source: NHS Scotland)", echo=FALSE}

expenses <- data.frame(
  Category=c("Staff costs", "Healthcare expenditures (drugs, supplies, etc.)", "Other supplies", "Other running costs (procurement, IT, HR, etc.)"),
  value=c(sample(25:35, size=1), sample(18:27, size=1), sample(4:10, size=1), sample(5:11, size=1))
)

ggplot(expenses, aes(x="", y=value, fill=Category)) +
  geom_bar(stat="identity", width=1, color="white") +
  coord_polar("y", start=0) +
  theme_void() 

```

The `r hospital_name` has units specialising in coronary care; maternity care; dermatology; gastroenterology; an ear, nose, and throat clinic; an eye clinic; and an orthopedic unit.

There are currently 25 ICU beds (with ventilators) available, an increase from the 14 ICU beds in the `r hospital_name` in November 2019. Average ICU bed occupancy before and during the COVID-19 pandemic is in the figure below. 


```{r A6, fig.cap="ICU bed occupancy for the Rosblane North Shore hospital (Source: NHS Scotland)", echo=FALSE}

ICU <- c(sample(10:15, size=16, replace=TRUE), sample(23:25, size=4, replace=TRUE), sample(17:20, size=4, replace=TRUE), sample(22:25, size=4, replace=TRUE), sample(15:20, size=5, replace=TRUE), sample(22:25, size=3, replace=TRUE), sample(15:20, size=4, replace=TRUE), sample(22:25, size=4, replace=TRUE), sample(17:21, size=4, replace=TRUE))
ICUdates <- c("1/2019", "2/2019", "3/2019", "4/2019", "5/2019", "6/2019", "7/2019", "8/2019", "9/2019", "10/2019", "11/2019", "12/2019", "1/2020", "2/2020", "3/2020", "4/2020", "5/2020", "6/2020", "7/2020", "8/2020", "9/2020", "10/2020", "11/2020", "12/2020", "1/2021", "2/2021", "3/2021", "4/2021", "5/2021", "6/2021", "7/2021", "8/2021", "9/2021", "10/2021", "11/2021", "12/2021","1/2022", "2/2022", "3/2022", "4/2022", "5/2022", "6/2022", "7/2022", "8/2022", "9/2022", "10/2022", "11/2022", "12/2022")

ICUdates <- factor(ICUdates, levels = ICUdates)

ICUstats <- data.frame(ICU, ICUdates)
ggplot(data = ICUstats, aes(x=ICUdates, y=ICU, group = 1)) + 
  geom_point() +
  geom_line() +
  labs(y = "Average North Shore ICU bed occupancy/month", x = "Month") +
  scale_y_continuous(limits=c(0, 25)) + 
  theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1)) + 
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())
```


There are approximately 1.3m admissions to the `r hospital_name` hospital annually, the majority of these being to the emergency (A&E) department.

```{r A7, fig.cap="Admissions to the Rosblane North Shore hospital by category (Source: NHS Scotland)", echo=FALSE}

admissions <- data.frame(
  Category=c("Elective Inpatient", "A&E", "Day case"),
  value=c(sample(15:25, size=1), sample(50:70, size=1), sample(20:25, size=1))
)

ggplot(admissions, aes(x="", y=value, fill=Category)) +
  geom_bar(stat="identity", width=1, color="white") +
  coord_polar("y", start=0) +
  theme_void() 

```


A&E waiting times have been called "disastrously high" in recent months, both by the press and by tthe hospital ombudsman.

```{r A8, fig.cap="ICU waiting times for the Rosblane North Shore hospital (Source: NHS Scotland)", echo=FALSE}

WaitTimes <- c(sample(1:2, size=16, replace=TRUE), sample(1:2, size=4, replace=TRUE), sample(2:4, size=4, replace=TRUE), sample(2:5, size=4, replace=TRUE), sample(1:5, size=5, replace=TRUE), sample(2:5, size=3, replace=TRUE), sample(3:5, size=4, replace=TRUE), sample(2:5, size=4, replace=TRUE), sample(5:6, size=4, replace=TRUE))
AEdates <- c("1/2019", "2/2019", "3/2019", "4/2019", "5/2019", "6/2019", "7/2019", "8/2019", "9/2019", "10/2019", "11/2019", "12/2019", "1/2020", "2/2020", "3/2020", "4/2020", "5/2020", "6/2020", "7/2020", "8/2020", "9/2020", "10/2020", "11/2020", "12/2020", "1/2021", "2/2021", "3/2021", "4/2021", "5/2021", "6/2021", "7/2021", "8/2021", "9/2021", "10/2021", "11/2021", "12/2021","1/2022", "2/2022", "3/2022", "4/2022", "5/2022", "6/2022", "7/2022", "8/2022", "9/2022", "10/2022", "11/2022", "12/2022")

AEdates <- factor(AEdates, levels = AEdates)

AEwaits <- data.frame(WaitTimes, AEdates)
ggplot(data = AEwaits, aes(x=AEdates, y=WaitTimes, group = 1)) + 
  geom_point() +
  geom_line() +
  labs(y = "Percentage of A&E patients with a wait time > 4 hours", x = "Month") +
  scale_y_continuous(limits=c(0, 15)) + 
  theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1)) + 
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())
```


The `r hospital_name` has been set a goal of 25% reduction in sepsis mortality (compared to past years)and 50% reduction in the nosocomial transmission of superbugs such as methicillin-resistant <i>Staphylococcus aureus</i> (MRSA), <i>Clostridiodes difficile</i> (C. diff), and <i>Candida auris</i> (C. auris). 

```{r A9, fig.cap="Key Performance Indicators (KPIs) related to infectious diseases for the Rosblane North Shore hospital (Source: NHS Scotland)", echo=FALSE}

hosKPIs <- data.frame(
  KPI=c("Reduction in MRSA cases", "Reduction in C diff cases", "Reduction in sepsis mortality", "Reduction in C auris cases"),
  percent=c(sample(55:65, size=1), sample(30:70, size=1), sample(5:15, size=1), sample(15:40, size=1))
)

ggplot(hosKPIs, aes(x = KPI, y = percent)) +
  geom_col() +
  coord_flip() +
  scale_y_continuous(limits=c(0, 100)) +
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())

```

There are currently `r sample(143:180, size=1)` patients in the `r hospital_name` with confirmed or suspected <i>Campylobacter jejuni</i>, with `r sample(15:17, size=1)` of these patients currently in the ICU. 

The hospital follows the guidelines set out in the [National Infection Prevention and Control Manual](https://www.nipcm.hps.scot.nhs.uk/).

As a hospital director, you are familiar with the literature surrounding hospital pandemic preparedness, and regularly read papers such as:

* Mer, Mervyn et al. “Critical Care Pandemic Preparation: Considerations and Lessons Learned from COVID-19.” Critical care clinics vol. 38,4 (2022): 761-774. 

* Tacconelli, Evelina et al. “Challenges of data sharing in European Covid-19 projects: A learning opportunity for advancing pandemic preparedness and response.” The Lancet regional health. Europe vol. 21 (2022): 100467. 

* Adelaja, I., Sayma, M., Walton, H., McLachlan, G., de Boisanger, J., Bartlett-Pestell, S., Roche, E., Gandhi, V., Wilson, G. J., Brookes, Z., Yeen Fung, C., Macfarlane, H., Navaratnam, A., James, C., Scolding, P., & Sara, H. (2020). A comprehensive hospital agile preparedness (CHAPs) tool for pandemic preparedness, based on the COVID-19 experience. Future healthcare journal, 7(2), 165–168. 

***

## Information Packet D5

You are a civil servant working in `r city_name`, working closely with the provost, city council and various agencies including the NHS, the UK Health Security Agency (UKHSA), and the Scottish Environment Protection Agency (SEPA).  

Some of the key `r city_name` facts and figures at a glance are summarized in the table below.

<table>
<table border="1">
  <tr>
    <th>Community Amenities</th>
    <th>Details</th>
  </tr>
  <tr>
    <td>Schools</td>
    <td>`r sample(2:8, size=1)` primary schools, `r sample(2:5, size=1)` secondary schools, 1 university</td>
  </tr>
  <tr>
    <td>Care Homes</td>
    <td>`r sample(1:5, size=1)`, each with ~`r sample(c(30, 40, 45, 50), size=1)` bed capacity</td>
  </tr>
  <tr>
    <td>Restaurants and Cafes</td>
    <td>`r sample(50:150, size=1)`</td>
  </tr>
    <tr>
    <td>Parks and Playgrounds</td>
    <td>`r sample(1:15, size=1)`</td>
  </tr>
  <tr>
  <th>Economic Indicators</th>
  <th>Details</th>
  </tr>
  <tr>
  <td>GDP/capita</td>
  <td>£`r sample(36000:50000, size=1)`</td>
  </tr>
  <tr>
  <td>Unemployment Rate</td>
  <td>`r sample(3:8, size=1)`%</td>
  </tr>
  <tr>
  <th>Traffic and Transport</th>
  <th>Details</th>
  </tr>
  <tr>
  <td>Traffic fatalities (average/month)</td>
  <td>`r sample(5:20, size=1)`</td>
  </tr>
</table>  
<br><br>


Your briefing packet on <i>Campylobacter jejuni</i> includes the following information.

* [Campylobacter information and guidance (Food standards Scotland)](https://www.foodstandards.gov.scot/downloads/Factsheet_-_Campylobacter_-_Website_-_for_businesses_and_professionals_-_June_2017.pdf)

* [Campylobacter "Fit for Travel" advice](https://www.fitfortravel.nhs.uk/advice/disease-prevention-advice/campylobacter)

* MacRitchie, L A et al. “Consumer acceptability of interventions to reduce <i>Campylobacter</i> in the poultry food chain.” Food control vol. 35,1 (2014): 260-266. doi:10.1016/j.foodcont.2013.06.005

* Bessell, Paul R et al. “Geographic determinants of reported human <i>Campylobacter</i> infections in Scotland.” BMC public health vol. 10 423. 15 Jul. 2010, doi:10.1186/1471-2458-10-423

* Scott, Elizabeth. “Food safety and foodborne disease in 21st century homes.” The Canadian journal of infectious diseases = Journal canadien des maladies infectieuses vol. 14,5 (2003): 277-80.

***

## Information Packet D6

You are a microbiologist working in the microbiology department at the Greater `r city_name` Clinical Laboratory. Your laboratory is equipped to handle a range of culture samples (including anaerobic and microaerobic bacteria, as well as viruses), and you routinely process`r sample(1500:2000, size=1)` samples monthly. 

You are familiar with the culture requirements for growth of <i>Campylobacter</i>, and the [UK Standards for Microbiology Investigations (UK SMI) for Identification of Campylobacter species](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/685065/ID_23i3.1.pdf).

```{r Campylobacter-agar, echo=FALSE, fig.cap="Campylobacter jejuni grown on Skirrow and Butzler growth medium. Image credit: [CDC](https://phil.cdc.gov/Details.aspx?pid=15986)"}
knitr::include_graphics(rep("images/Campy-plates.jpg"))
```

You are also familiar with the literature surrounding <i>Campylobacter</i> identification, and often read papers such as:

* LaGier, Michael J et al. “A real-time multiplexed PCR assay for rapid detection and differentiation of <i>Campylobacter jejuni</i> and <i>Campylobacter coli</i>.” Molecular and cellular probes vol. 18,4 (2004): 275-82.

* Ferrari, Sevinc et al. “Validation of PCR methods for confirmation and species identification of thermotolerant <i>Campylobacter</i> as part of EN ISO 10272 - Microbiology of the food chain - Horizontal method for detection and enumeration of <i>Campylobacter</i> spp.” International journal of food microbiology vol. 388 (2023): 110064.

* Ojima-Kato, Teruyo et al. “Proteotyping of <i>Campylobacter jejuni</i> by MALDI-TOF MS and Strain Solution Version 2 Software.” Microorganisms vol. 11,1 202. 12 Jan. 2023, doi:10.3390/microorganisms11010202

* Hong, Seung-Hwan et al. “Gold Nanoparticle and Polymerase Chain Reaction (PCR)-Based Colorimetric Assay for the Identification of <i>Campylobacter</i> spp. in Chicken Carcass.” Food science of animal resources vol. 43,1 (2023): 73-84. doi:10.5851/kosfa.2022.e59

***

## Information Packet D7

You are a public communications expert, often employed by the `r city_name` city council to manage the PR response to different initiatives or crises. 

As a public communications expert, you are familiar with the literature surrounding science communication in a pandemic, including papers such as: 

* Matta, G. Science communication as a preventative tool in the COVID19 pandemic. Humanit Soc Sci Commun 7, 159 (2020).

* Abdool Karim, Salim S. “Public understanding of science: Communicating in the midst of a pandemic.” Public understanding of science (Bristol, England) vol. 31,3 (2022): 282-287.

* Royan, Regina et al. “Use of Twitter Amplifiers by Medical Professionals to Combat Misinformation During the COVID-19 Pandemic.” Journal of medical Internet research vol. 24,7 e38324. 22 Jul. 2022, doi:10.2196/38324

* Tait, Margaret E et al. “Serving the public? A content analysis of COVID-19 public service announcements airing from March - December of 2020 in the U.S.” Preventive medicine reports vol. 29 (2022): 101971. 

You have assembled the following information/resources for this meeting:

* [World Food Safety Day Campaign Materials](https://www.who.int/campaigns/world-food-safety-day/2022/campaign-materials)

* [Campylobacter campaign partner toolkit](https://www.foodstandards.gov.scot/publications-and-research/publications/campylobacter-campaign-partner-toolkit)

* Gölz, Greta et al. “Relevance of <i>Campylobacter</i> to public health--the need for a One Health approach.” International journal of medical microbiology : IJMM vol. 304,7 (2014): 817-23.

***

## Information Packet D8

You are one of the doctors at the largest surgery in `r city_name`. In the past fortnight, `r sample(50:100, size=1)` of your patients have recently presented to the surgery with symptoms consistent with <i>Campylobacter jejuni</i>. You have therefore recently been familiarising yourself with the relevant literature:

* [Campylobacter Information for Health Professionals (CDC)](https://www.cdc.gov/campylobacter/technical.html)

* [BMJ Best Practice - Campylobacter Infection](https://bestpractice.bmj.com/topics/en-gb/1175)

* [NICE Gastroenteritis Background Information](https://cks.nice.org.uk/topics/gastroenteritis/background-information/causes/)

* [NICE Gastroenteritis: Management](https://cks.nice.org.uk/topics/gastroenteritis/management/)

* Tang, Yizhi et al. “Antibiotic resistance trends and mechanisms in the foodborne pathogen, <i>Campylobacter</i>.” Animal health research reviews vol. 18,2 (2017): 87-98.

* Valenčak-Ignjatić, Ivana et al. “<i>Campylobacter jejuni</i> subdural hygroma infection in a 2-year old boy: case report and a brief literature review.” BMC infectious diseases vol. 22,1 700. 20 Aug. 2022, doi:10.1186/s12879-022-07680-0

* Lévesque, Simon et al. “Campylobacteriosis in urban versus rural areas: a case-case study integrated with molecular typing to validate risk factors and to attribute sources of infection.” PloS one vol. 8,12 e83731. 26 Dec. 2013, doi:10.1371/journal.pone.0083731

***

## Information Packet D9

You are an epidemiologist working at the University of `r city_name`, and have been studying <i>Campylobacter</i> transmission dynamics for the past `r sample(2:10, size=1)` years. As such, you are very familiar with the relevant literature, including papers such as: 

* Bishop, H et al. “Bacteriological Survey of Fresh Minced Beef on Sale at Retail Outlets in Scotland in 2019: Three Foodborne Pathogens, Hygiene Process Indicators, and Phenotypic Antimicrobial Resistance.” Journal of food protection vol. 85,9 (2022): 1370-1379. 

* Lopes, Bruno S et al. “Nationwide Stepwise Emergence and Evolution of Multidrug-Resistant <i>Campylobacter jejuni</i> Sequence Type 5136, United Kingdom.” Emerging infectious diseases vol. 25,7 (2019): 1320-1329. 

* Sheppard, Samuel K et al. “Evolution of an agriculture-associated disease causing <i>Campylobacter coli</i> clade: evidence from national surveillance data in Scotland.” PloS one vol. 5,12 e15708. 15 Dec. 2010, doi:10.1371/journal.pone.0015708

* Sheppard, Samuel K et al. “<i>Campylobacter</i> genotypes from food animals, environmental sources and clinical disease in Scotland 2005/6.” International journal of food microbiology vol. 134,1-2 (2009): 96-103.

* Dessouky, Yara El et al. “Genomic insights into zoonotic transmission and antimicrobial resistance in <i>Campylobacter jejuni</i> from farm to fork: a one health perspective.” Gut pathogens vol. 14,1 44. 5 Dec. 2022, doi:10.1186/s13099-022-00517-w

* Hadiyan, Maryam et al. “Prevalence, antimicrobial resistance, virulence gene profile and molecular typing of <i>Campylobacter</i> species isolated from poultry meat samples.” Veterinary medicine and science vol. 8,6 (2022): 2482-2493. 

* Ocejo M, Oporto B, Lavín JL, Hurtado A. Monitoring within-farm transmission dynamics of antimicrobial-resistant Campylobacter in dairy cattle using broth microdilution and long-read whole genome sequencing. Sci Rep. 2023;13(1):12529. Published 2023 Aug 2. doi:10.1038/s41598-023-39588-3

* Rodrigues JA, Blankenship HM, Cha W, et al. Pangenomic analyses of antibiotic-resistant Campylobacter jejuni reveal unique lineage distributions and epidemiological associations. Microb Genom. 2023;9(8):mgen001073. doi:10.1099/mgen.0.001073

* Katz A, Porte L, Weitzel T, et al. Whole-genome sequencing reveals changes in genomic diversity and distinctive repertoires of T3SS and T6SS effector candidates in Chilean clinical Campylobacter strains. Front Cell Infect Microbiol. 2023;13:1208825. Published 2023 Jul 13. doi:10.3389/fcimb.2023.1208825
